index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Carrie Short - Web Developer</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link type="text/css" rel="stylesheet" href="vendor/styles/font-awesome.min.css">
+    <link type="text/css" rel="stylesheet" href="styles/base/reset.css">
+    <link type="text/css" rel="stylesheet" href="styles/base/default.css">
+    <link type="text/css" rel="stylesheet" href="styles/layout.css">
+    <link type="text/css" rel="stylesheet" href="styles/module/header.css">
+    <link type="text/css" rel="stylesheet" href="styles/module/banner.css">
+    <link type="text/css" rel="stylesheet" href="styles/module/portfolio-preview.css">
+    <link type="text/css" rel="stylesheet" href="styles/state/hover.css">
+    <link type="text/css" rel="stylesheet" href="styles/state/sticky.css">
+    <script id="preview-template" type="text/x-handlebars-template">
+        <article class="project-preview" data-project="{{project}}" style="background-image:url(img/projects/{{previewImage}})">
```

Comment:
If you store the {{previewImage}} in a data attribute you could access the unique vale and update the background through css using that value instead of defining inline style tag.